app/models/study.rb

```diff
@@ -764,28 +764,34 @@ def can_compute?(user)
     if user.nil? || !user.registered_for_firecloud?
       false
     else
-      begin
-        workspace_acl = Study.firecloud_client.get_workspace_acl(self.firecloud_project, self.firecloud_workspace)
-        if workspace_acl['acl'][user.email].nil?
-          # check if user has project-level permissions
-          user_client = FireCloudClient.new(user, self.firecloud_project)
-          projects = user_client.get_billing_projects
-          # billing project users can only create workspaces, so unless user is an owner, user cannot compute
-          projects.detect {|project| project['projectName'] == self.firecloud_project && project['role'] == 'Owner'}.present?
-        else
-          workspace_acl['acl'][user.email]['canCompute']
+      # don't check permissions if API is not 'ok'
+      if Study.firecloud_client.services_available?('Sam', 'Rawls', 'Agora')
```

Comment:
do you think we should make these strings constants somewhere?  e.g.  FireCloudClient::SAM_SERVICE = 'Sam'.

Reply:
That's actually a great idea.  It should be there, and comments to the effect of what service covers what.  That data only lives in my brain and our playbook... it would be great to have them in code.

Reply:
555b181